api-spec/protobuf/operator.proto

```diff
@@ -7,87 +7,90 @@ import "types.proto";
+  rpc ListFeeAddresses(ListFeeAddressesRequest)
+    returns (ListFeeAddressesReply) {}
 
-  // Returns the aggregated balance of LBTC held in the fee account.
-  rpc BalanceFeeAccount(BalanceFeeAccountRequest)
-      returns (BalanceFeeAccountReply) {}
+  // Returns info about the balance of LBTC held in the fee account.
+  rpc GetFeeBalance(GetFeeBalanceRequest) returns (GetFeeBalanceReply) {}
 
-    // Operator can provide transaction(s) outpoints of deposits made to fund a new market.
-  // The transaction must be visible and confirmed, otherwise an error will be returned,
-  // inviting the operator to retry again
-  rpc ClaimMarketDeposit(ClaimMarketDepositRequest) returns (ClaimMarketDepositReply) {}
+  // Allows to provide transaction(s) outpoints of deposits made to fund the fee account.
+  // The transaction(s) must be already included in blockchain.
+  rpc ClaimFeeDeposits(ClaimFeeDepositsRequest) returns (ClaimFeeDepositsReply) {}
 
+  // Allows to withdraw funds from the fee account to a given address.
+  rpc WithdrawFee(WithdrawFeeRequest) returns (WithdrawFeeReply) {}
+
+  // Creates a new market account in the daemon's wallet.
+  rpc NewMarket(NewMarketRequest) returns (NewMarketReply) {}
   
-  // Operator can provide transaction(s) outpoints of deposits made to fund the fee account.
-  // The transaction must be visible and confirmed, otherwise an error will be returned,
-  // inviting the operator to retry again
-  rpc ClaimFeeDeposit(ClaimFeeDepositRequest) returns (ClaimFeeDepositReply) {}
+  // Returns some new derived address(es) for the given market.
+  rpc GetMarketAddress(GetMarketAddressRequest)
+    returns (GetMarketAddressReply) {}
+
+  // Returns the list of all derived addresses for the given market.
+  rpc ListMarketAddresses(ListMarketAddressesRequest)
+    returns (ListMarketAddressesReply) {}
 
-  // Makes the given market tradable
+  // Returns info about the balance of the given market.
+  rpc GetMarketBalance(GetMarketBalanceRequest) returns (GetMarketBalanceReply) {}
+
+  // Allows to provide transaction(s) outpoints of deposits made to fund a market.
+  // The transaction(s) provided must be already included in blockchain.
+  rpc ClaimMarketDeposits(ClaimMarketDepositsRequest) returns (ClaimMarketDepositsReply) {}
+
+  // Makes the given market tradable.
   rpc OpenMarket(OpenMarketRequest) returns (OpenMarketReply) {}
-  // Makes the given market NOT tradabale
+  
+  // Makes the given market NOT tradabale.
   rpc CloseMarket(CloseMarketRequest) returns (CloseMarketReply) {}
 
-  // Get extended details for each markets either open, closed or to be funded.
-  rpc ListMarket(ListMarketRequest) returns (ListMarketReply) {}
+  // Deletes a market.
+  rpc DropMarket(DropMarketRequest) returns(DropMarketReply){}
+
+  // Displays a report of the colected fees for the given market.
+  rpc ReportMarketFee(ReportMarketFeeRequest) returns (ReportMarketFeeReply) {}
```

Comment:
I would modify also this RPC to make it more clear which `fee` we are talking about, to not clash with `UpdateMarket{Percentage|Fixed}Fee` meaning 

```suggestion
  rpc ReportSwapFeesForMarket(ReportMarketFeeRequest) returns (ReportMarketFeeReply) {}
```